zerver/models.py

```diff
@@ -3313,6 +3313,23 @@ def increment_times_used(self) -> None:
         self.save(update_fields=["times_used"])
 
 
+class MissedMessageEmailEntry(models.Model):
+    """
+    Store the missedmessage events for a user which are yet to be processed.
+    """
+
+    user_profile: UserProfile = models.ForeignKey(UserProfile, on_delete=CASCADE)
+
+    message: Message = models.ForeignKey(Message, on_delete=CASCADE)
+    trigger: str = models.TextField(max_length=20)
+    mentioned_user_group: Optional[UserGroup] = models.ForeignKey(
+        UserGroup, null=True, on_delete=CASCADE
+    )
+
+    # Calculated from the time event was received and the batching period.
+    expiry_time: datetime.datetime = models.DateTimeField(db_index=True)
```

Comment:
Maybe `scheduled_timestamp` would be appropriate?  I think it has very similar semantics to the fields of that name in `ScheduledEmail` and `ScheduledMessage`.

Reply:
Done